packages/ui-extensions/src/surfaces/checkout/components/Badge/content/guidelines.md

```diff
@@ -0,0 +1,8 @@
+## Best practices
+
+- Aim for 1 word per badge.
+- For complex states that require 2 words, use sentence case.
+- For status badge labels, use an adjective (for example, “Available” or “Complete”) or a verb in the past tense (for example, “Delivered” or “Delayed”)
+- Use the tone prop to indicate the level of importance. `subdued` provides the least emphasis, while `critical` indicates the highest level of urgency.
+- Write a useful `accessibilityLabel` so that customers using assistive technology can access the full meaning of the badge in context. For badges with the `critical` tone, include information that conveys the urgency of the badge (for example, “Warning” or “Alert”).
```

Comment:
It may be good to emphasis that it's only needed if the badge content is not descriptive enough for non-sighted users that can't view the whole context in one shot.

I'm trying to ensure developers that are uncertain or unaware of a11y best practices to **not** fill the `accessibilityLabel` with the same content as the Badge. I see that often on Button (ex: `<Button accessibilityLabel="submit">submit</Button>`). This duplicates announcements and ends up creating a poor experience for screen-reader users.